hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/InitialModServiceTokenSchema.java

```diff
@@ -145,7 +146,7 @@ public InitialModServiceTokenSchema(
     @Override
     public Set<StateDefinition> statesToCreate() {
         return Set.of(
-                StateDefinition.inMemory(TOKENS_KEY, TokenID.PROTOBUF, Token.PROTOBUF),
+                StateDefinition.onDisk(TOKENS_KEY, TokenID.PROTOBUF, Token.PROTOBUF, MAX_TOKENS),
```

Comment:
Only `tokenRels` are on disk and not tokens as far as I know from [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesState.java#L526)

Reply:
Discussed as suggestion from Oleg.